common/config/rush/browser-approved-packages.json

```diff
@@ -418,6 +418,10 @@
       "name": "cache-require-paths",
       "allowedCategories": [ "internal" ]
     },
+    {
+      "name": "canonical-path",
```

Comment:
Where is this used?

Reply:
used in webserver.ts, was removed for some changes, but added usage back now